octoprint_mrbeam/static/js/material-store.js

```diff
@@ -65,17 +66,19 @@ $(function () {
         };
 
         self.sendMessageToMaterialStoreIframe = function (event, payload) {
+            let materialStoreIframeElement = $("#material_store_iframe");
             let data = {
                 event: event,
                 payload: payload,
             };
 
-            document
-                .getElementById("material_store_iframe")
-                .contentWindow.postMessage(
-                    data,
-                    self.material_store_iframe_src
-                );
+            if(materialStoreIframeElement.is(":visible")){
```

Comment:
but if you are calling this method the iframe should be visible right? otherwise you shouldn't call this method... I think it's ok to break here if it's not visible, so that we catch the bug as soon as possible... at least an else  bracket console logging the error

Reply:
resolved